Analyses/src/BFieldPlotter_module.cc

```diff
@@ -159,23 +183,35 @@ namespace mu2e {
 	  axisTwo += (binTwo) ? -_mapBinSize/100. : _mapBinSize/100.;
 	CLHEP::Hep3Vector point;
 	CLHEP::Hep3Vector field;
-	if(_plane == "x") 
+	if(_plane == "x")
 	  point = CLHEP::Hep3Vector(_planeValue, axisOne, axisTwo);
-	else if(_plane == "y") 
+	else if(_plane == "y")
 	  point = CLHEP::Hep3Vector(axisOne, _planeValue, axisTwo);
-	else if(_plane == "z") 
+	else if(_plane == "z")
 	  point = CLHEP::Hep3Vector(axisOne, axisTwo, _planeValue);
 	if(map)
 	  map->getBFieldWithStatus(point,field);
 	else //if null, get the manager and plot the field returned by it
 	  bf->getBFieldWithStatus(point,field);
 
 	_hMap[name]->Fill(axisOne, axisTwo, field.mag()); //fill with weight of the field magnitude
+
+	if(_detector) point = det->toDetector(point);
```

Comment:
Redefining  "point" here may be a landmine for the future.   If one adds new code using "point" above or below this line things will behave differently.  Even worse, the breakage depends on specific job config.
Please use a new variable like
const CLHEP::Hep3Vector dumpPoint = detector_ ? point : det->toDetector(point);

Reply:
OK.